CMakeLists.txt

```diff
@@ -169,21 +170,60 @@ include_directories(SYSTEM
 
 
 #
+# Set linker (ld) 
+# use ld.gold if available
+#
+execute_process(COMMAND ld.gold --version
```

Comment:
@david-ragazzi similar, how do I check for availability of the "gold linker"? I only found 
https://stackoverflow.com/questions/9369827/how-do-i-specify-the-linker-when-building-python-extensions
Thanks!